documentation/puppet-api/v3/environment_classes.md

```diff
@@ -370,18 +366,17 @@ environment's class information.
 
 #### Clearing class information cache entries
 
-After updating an environment's manifests, the Server's class information cache entries
-must be cleared so the server can parse the latest manifests and reflect class changes to
-the class information in later queries to the environment classes endpoint. To clear cache
+After updating an environment's manifests, you must clear the server's class information cache entries, so the server can parse the latest manifests and reflect class changes to
+the class information in queries to the environment classes endpoint. To clear cache
 entries on the server, do one of the following:
 
-1.  Call the
-    [`environment-cache` API endpoint][environment cache API].
+-   Call the
+    [`environment-cache` API endpoint](/puppet-server/latest/admin-api/v1/environment-cache.html).
```

Comment:
I think this needs to be `/puppetserver/latest/admin-api/...` or actually I think that `[environment cache API]` is defined and this doesn't need to be changed (except for the number -> bullet point).